src/annotations/AnnotationParser.php

```diff
@@ -167,6 +178,57 @@ public function parse($source, $path)
                     if ($type == T_FUNCTION) {
                         $state = self::METHOD_NAME;
                     }
+                    if ($type == T_USE) {
+                        $state = self::CLASS_USE_CLAUSE;
+                        $use = '';
+                    }
+                    break;
+
+                case self::CLASS_USE_CLAUSE:
+                    if ($type === self::CHAR) {
+                        if ($str === ';') {
+                            $state = self::SCAN_CLASS;
+                        } elseif ($str === '{') {
+                            $use = '';
+                            $state = self::CLASS_USE_BLOCK;
+                        }
+                    }
+                    break;
+
+                case self::CLASS_USE_BLOCK:
+                    if ($type == T_STRING || $type == T_NS_SEPARATOR || $type == T_DOUBLE_COLON) {
+                        $use .= $str;
+                    } elseif ($type === T_INSTEADOF) {
+                        $use_insteadof = '';
+                        $state = self::CLASS_USE_INSTEADOF;
+                    } elseif ($type === T_AS) {
+                        $use_as = '';
+                        $state = self::CLASS_USE_AS;
+                    } elseif ($type === self::CHAR) {
+                        if ($str === ';') {
+                            $use = '';
+                        } elseif ($str === '}') {
+                            $state = self::SCAN_CLASS;
+                        }
+                    }
+                    break;
+
+                case self::CLASS_USE_INSTEADOF:
+                    if ($type === self::CHAR && $str === ';') {
+                        $traitMethodOverrides[$class][substr($use, 2 + strrpos($use, '::'))] = $use;
```

Comment:
Just matching style here: https://github.com/benesch/php-annotations/blob/trait-docblocks/src/annotations/AnnotationParser.php#L135

(But that'll be blown away by #105 anyway, so will fix!.)